PYME/cluster/clusterUI/localization/views.py

```diff
@@ -145,7 +150,8 @@ def localize(request, analysisModule='LatGaussFitFR'):
         for seriesName in seriesToLaunch:
             try:
                 if USE_RULES:
-                    HTTPRulePusher.launch_localize(analysisMDH, seriesName)
+                    context = {'seriesName': seriesName}
+                    rule_factory.get_rule(context=context).push()
```

Comment:
this is being called incorrectly - should be `rule_factory.get_rule(seriesName=seriesName).push()`

Reply:
Pretty sure that'd throw a TypeError unexpected kwarg since `get_rule` only takes `self` and `context`

Reply:
oops, you're right. Does raise the question of what the behaviour should be ....

Reply:
ie: would it make more sense for the signature of `get_rule(self, context)` to be `get_rule(self, **context)`

Reply:
can we merge @David-Baddeley ? I don't personally care what the get_rule signature is, but if you change it we should update acquire/ui/rules as well